multimodal/src/autogluon/multimodal/matcher.py

```diff
@@ -1144,62 +1255,68 @@ def evaluate(
         else:
             return results
 
-    def semantic_search(
+    def evaluate(
         self,
-        query_data: Union[pd.DataFrame, dict, list],
-        response_data: Union[pd.DataFrame, dict, list],
-        id_mappings: Dict[str, Dict],
-        return_prob: Optional[bool] = False,
-        as_pandas: Optional[bool] = None,
+        data: Optional[Union[pd.DataFrame, dict, list]] = None,
+        query_data: Optional[list] = None,
+        response_data: Optional[list] = None,
+        id_mappings: Optional[Dict[str, Dict]] = None,
+        metrics: Optional[Union[str, List[str]]] = None,
+        return_pred: Optional[bool] = False,
+        chunk_size: Optional[int] = 1024,
+        cosine: Optional[bool] = True,
+        top_k: Optional[int] = 100,
     ):
         """
-        Perform a cosine similarity search between query data and response data.
+        Evaluate model on a test dataset.
 
         Parameters
         ----------
+        data
+            A dataframe, containing the same columns as the training data.
```

Comment:
We may need to state that it's relevance data. And put an example as below:

```python
| query_col1  | query_col2 | query_col3 | response_col1  | response_col2 | 
|-------------|------------|------------|----------------|---------------|
|             | ....       | ....       | ...            | ...           |
|             | ....       | ....       | ...            | ...           |
```

Reply:
Thanks. Added the example.